modules/api1/docs.md

```diff
@@ -33,6 +33,17 @@ In order to get particular project, use the following request:
 > GET [/projects/1](/en/api/1.0/projects/1)
 
 In the above `1` is the project ID.
+
+### Update project <a href="#projects-update" id="projects-update">#</a>
+
+In order to update project, use the following request: 
+
+> PUT [/projects/1](/en/api/1.0/projects)
+
+- primary_image_id: id of the primary image.
```

Comment:
Yes, also available: 'title', 'url', 'is_opensource', 'source_url', 'yii_version', 'description', 'status', 'tagValues'.

I can add them to docs or create a scenario for update via API.